webapp/test/test_updates.py

```diff
@@ -169,3 +169,72 @@ def test_process_list_empty_json(self):
             # NOTE: use copy of dict with json input, because process_list changes this dict
             self.updates_api.process_list(1, UPDATES_JSON_EMPTY.copy())
         assert "'package_list' is a required property" in str(context)
+
+class TestSplayTree(TestBase):
```

Comment:
On the top of file is the comment `# TODO: write unit tests for Hot Cache`. Isn't it prepared for these tests?